targetFunctions/+standardTF/layersCore.m

```diff
@@ -1,5 +1,5 @@
 function [sldProfile,reflect,Simul,shifted_dat,theseLayers,resamLayers,chiSq,ssubs] = ...
-    standardTFLayersCore(contrastLayers, rough, ...
+    layersCore(contrastLayers, rough, ...
```

Comment:
Given this routine is called by both standardTF and domainsTF layers calculations, I think it is appropriate to remove this function from the standardTF namespace and include it elsewhere.

Reply:
Also, perhaps the name "coreLayersCalculation" is more helpfully descriptive.

Reply:
My understanding is that domains, runs a couple of standard layers calculation then averages so the coreLayersCalculation belongs to standardTF even though it is used by domains. So I will rename the file but leave it in the same folder